ansible-tower/README.md

```diff
@@ -0,0 +1,85 @@
+## Prerequisistes:
+- [Docker](https://docs.docker.com/desktop/)
+- [Docker Compose](https://docs.docker.com/compose/install/)
+- [Terraform](https://www.terraform.io/downloads.html)
+- An AWS Subnet, preferably in us-west-2
+
+## Usage
+Fork this repo. Your fork will serve as a Repo Containing your infrastructure as code.
+
+Set the `REPO_URL` environment variable and clone your fork. Use HTTPS (not SSH) because Tower uses this url
+```
+export REPO_URL=https://github.com/domenicbove/demo-scene
```

Comment:
@domenicbove this URL would need to change.

Reply:
nice catch

Reply:
updated